### DISCUSSION TITLE
What is the optimal model and image size for training to detect small objects.

### DISCUSSION THREAD

**Discussion by Zypherous123** [ultralytics/yolov5#6083] [Q&A] [answered]
Hi, I want to train a model that's is used to detect relatively small and overlapping objects. What model and image size would you recommend to be used for training?

Would like to hear your recommendation.

Thanks

**Comment by glenn-jocher** [comment, marked as answer]
@Zypherous123 the P6 models mainly improve large object detection. Large image sizes are recommended for small object detection.

**Reply by Zypherous123** [reply]
Thanks for the reply. Is using YOLOv5l6 with image 1280 be an optimal trade off in terms of speed to performance ratio?

**Reply by Zypherous123** [reply]
My aim is to have high detection accuracy.
Since I will only be processing images, real time detection / speed is not an issue.

**Reply by glenn-jocher** [reply]
@Zypherous123 yeah YOLOv5l6 at 1280 is the standard training size for our official model on COCO, but remember COCO contains large objects also, i.e. 1000 pixels at 1280. If at 1280 all of your objects are small < 30 pixels you can probably just use YOLOv5l. In any case the P6 model won't hurt, it's a great place to start.

**Reply by Zypherous123** [reply]
Thank you for the insight. Great work on the model. Have heard good thing about it.
